feat(webhook): Add functioning conversion webhook with cert rotation (#1066)
stackabletech/operator-rs: crates/stackable-operator/CHANGELOG.md

### Changed

- Update `kube` to `1.1.0` ([#1049]).

> - BREAKING: Return type for `ListenerOperatorVolumeSourceBuilder::new()` is no longer a `Result` ([#1058]).

Reviewer: note: This is already released and as such needs to be removed here.

> - BREAKING: Require two new CLI arguments: `--operator-namespace` and `-operator-service-name`.
>   These are required, so that the operator knows what Service it needs to enter as CRD conversion webhook ([#1066]).

Reviewer: note: This is an addition and as such needs to be moved to the "Added" section. Also, slight reword:

```suggestion
- BREAKING: Add two new required CLI arguments: `--operator-namespace` and `-operator-service-name`.
  These two values are used to construct the service name in the CRD conversion webhook ([#1066]).
```

> - BREAKING: The `ProductOperatorRun` used for CLI arguments has some field renamed for consistency ([#1066]):
>   - `telemetry_arguments` -> `telemetry`
>   - `cluster_info_opts` -> `cluster_info`

Reviewer: suggestion: Slight reword.

```suggestion
- BREAKING: Rename two fields of the `ProductOperatorRun` struct for consistency and clarity ([#1066]):
  - `telemetry_arguments` -> `telemetry`
  - `cluster_info_opts` -> `cluster_info`
```
